### PR TITLE
[WIP] Feature/mklnn default make + test on jenkins

### PR DESCRIPTION
**Pull request by azai91**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -132,7 +136,7 @@ ifeq ($(USE_MKLDNN), 1)
 		LDFLAGS += -L$(MKLROOT)/lib
 	endif
 	CFLAGS += -I$(MKLDNNROOT)/include
-	LDFLAGS += -L$(MKLDNNROOT)/lib -lmkldnn -Wl,-rpath,'$${ORIGIN}'
+	LDFLAGS += -L$(MKLDNNROOT)/lib -lmkldnn -Wl,-rpath,$(ROOTDIR)/lib
```
Comment (szha) [inline Makefile]: You want a hard-coded path as rpath?

**Comment by azai91** [comment]
@iblis17 can you take a look at the julia build and see why it fails when MKLDNN is enabled?

**Comment by iblis17** [comment]
The log looks weird:
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-cpu/detail/PR-13464/40/pipeline#step-500-log-223

```
+ export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so
+ LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so
+ julia -e 'Pkg.build("MXNet")'
```

It doesn't match the code in this PR
```
    export LD_PRELOAD='/usr/lib/x86_64-linux-gnu/libjemalloc.so'
    export LD_LIBRARY_PATH=/work/mxnet/lib:$LD_LIBRARY_PATH
    # use the prebuilt binary from $MXNET_HOME/lib
    julia -e 'Pkg.build("MXNet")'
```

The line `LD_LIBRARY_PATH` is gone. Any idea?

**Comment by mseth10** [comment]
This PR is deprecated. The new PR is #13681. @azai91 can you please close this PR. Thanks!

**Pull request closed without merging**